app/models/conference.rb

```diff
@@ -329,8 +329,10 @@ def event_distribution
   # * +hash+ -> hash
   def scheduled_event_distribution
     confirmed_events = program.events.where(state: 'confirmed')
-    scheduled_value =  { 'value' => confirmed_events.where.not(start_time: nil).count, 'color' => 'green' }
-    unscheduled_value =  { 'value' => confirmed_events.where(start_time: nil).count, 'color' => 'red' }
+    confirmed_scheduled_events = confirmed_events.select{|event| event.scheduled?}
```

Comment:
Yes we should surely add this, but using it here might make the code a bit confusing.It will give all the scheduled events from all the conferences so we will have to select only the events of that particular conference.Ideal solution for me would be something like this:

`program.events.scheduled`

what do you think?